infrastructure/apim-nrl.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_api_gateway_usage_plan" "apim" {
+  name = "${terraform.workspace}_apim_usage_plan"
```

Comment:
```suggestion
resource "aws_api_gateway_usage_plan" "apim" {
  name = "${terraform.workspace}_apim-usage-plan"
```

Reply:
Should we mix _ and -?

Reply:
@NogaNHS Good point do we not use - everywhere else? 
<img width="383" alt="image" src="https://github.com/user-attachments/assets/ba348c3a-3820-48fd-bee3-20c1b5ba5516">

Reply:
Yes, let's not mix and just use `-`